src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  const title = e.target.closest('li');
+  const childs = [...title.children];
+
+  for (const el of childs) {
+    if (el.style.display === 'none') {
+      el.style.display = 'block';
+      title.style.fontWeight = 'normal';
```

Comment:
The logic for setting `title.style.fontWeight` to 'normal' or 'bold' seems inverted. Typically, when elements are shown, the title should be bold to indicate expanded state, and normal when collapsed. Consider revising this logic if it doesn't match the intended behavior.